modules/apps/account/account-test/src/testIntegration/java/com/liferay/account/internal/exportimport/data/handler/test/AccountEntriesAdminPortletDataHandlerTest.java

```diff
@@ -225,6 +235,135 @@ public void testExportImportAccountEntries() throws Exception {
 						ExportImportReportEntryConstants.TYPE_EMPTY)));
 	}
 
+	@Test
+	@TestInfo("LPD-64512")
+	public void testExportImportAccountEntriesWithError() throws Exception {
```

Comment:
What do you think about doing the test of throwing an exception and registering the Report Entry in `BatchTestEntityExportImportTest` instead of here? I have been checking that class and we are already validating there the report entry is being created if the `ImportTaskPreAction` fails.

We could do the same for the `ItemReaderPostAction`. What do you think?

Reply:
I think we are using that module to avoid creating tests to test the framework in concrete modules